Formula/step.rb

```diff
@@ -15,8 +15,8 @@ class Step < Formula
   depends_on "go" => :build
 
   resource "certificates" do
-    url "https://github.com/smallstep/certificates/releases/download/v0.13.2/step-certificates_0.13.2.tar.gz"
-    sha256 "433e0c040a7161ae767db959528c80db365f74401da0338ef28aeaf748267f06"
+    url "https://github.com/smallstep/certificates/releases/download/v0.13.3/step-certificates_0.13.3.tar.gz"
+    sha256 "a198e0853f36c775392fbb37da1e986e66572270800413dd70d76b31a963158c<Paste>"
```

Comment:
@dopey It looks like the sha is failing in the build. Accidental `<Paste>` text?

```
00:21:56 Warning: Resource certificates reports different SHA256: a198e0853f36c775392fbb37da1e986e66572270800413dd70d76b31a963158c<Paste>
00:21:56 Downloaded to: /Users/brew/Library/Caches/Homebrew/downloads/7aa9f884b7896b20b97a972883342063eb9d7ca18a81494940ac3b8854126461--step-certificates_0.13.3.tar.gz
00:21:56 SHA256: a198e0853f36c775392fbb37da1e986e66572270800413dd70d76b31a963158c
```

https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/version=catalina/49015/console

Reply:
fixed. oof